packages/react-tree-grid/package.json

```diff
@@ -11,8 +11,12 @@
   },
   "dependencies": {
     "@fluentui/react-context-selector": ">=9.1.47 < 10.0.0",
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
     "@fluentui/keyboard-keys": ">=9.0.6 < 10.0.0",
     "@fluentui/react-tabster": ">=9.14.0 < 10.0.0",
     "@fluentui/react-utilities": ">=9.16.0 < 10.0.0"
+  },
+  "devDependencies": {
+    "react-window": "^1.8.10"
```

Comment:
why is this needed ? package.json shouldn't contain any `devDependendies` definitions as that has nothing to do with production release.